apps/web/src/styles.css

```
@@ -2360,50 +2642,100 @@ p {
+.calendar-time-axis li:not([data-major="true"]) {
+  color: color-mix(in srgb, var(--muted) 72%, transparent);
+  font-size: 7px;
+}
+.calendar-time-axis li[data-major="true"] {
+  font-weight: 650;
+}
 .calendar-timeline {
+  --calendar-hour-rule: var(--line);
+  --calendar-half-hour-rule: color-mix(in srgb, var(--line) 72%, transparent);
+  --calendar-quarter-hour-rule: color-mix(in srgb, var(--line) 42%, transparent);
   background-color: var(--surface-raised);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)